engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRProvider.java

```diff
@@ -106,6 +113,12 @@ public boolean isInitialized() {
     }
 
     public void shutdown() {
+        JOpenVRLibrary.VR_ShutdownInternal();
```

Comment:
I forgot: who is managing the FBOs VR uses? Who disposes them? One of the existing FBO managers or the VR code? If so, should FBO disposal happen here?